src/styles/styles/styles.android.js

```diff
@@ -0,0 +1,38 @@
+import variables from '../variables';
+import baseStyles from './styles.native';
+
+const styles = {
+    ...baseStyles.styles,
```

Comment:
The `baseStyles` object does not have a `styles` property AFAICT

```suggestion
```

Reply:
Got you, would remove this